source/_components/vacuum.template.markdown

```diff
@@ -15,16 +15,24 @@ return_to_base, clean_spot, locate and set_fan_speed commands of a vacuum.
 To enable Template Vacuums in your installation, add the following to your
 `configuration.yaml` file:
 
+{% raw %}
+
 ```yaml
 # Example configuration.yaml entry
 vacuum:
   - platform: template
     vacuums:
       living_room_vacuum:
         start:
-            service: script.vacuum_start
+          service: script.vacuum_start
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.